.github/workflows/Rbuild.yml

```diff
@@ -0,0 +1,12 @@
+name: R Build and Checks
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+    - name: R Build and Checks
+      uses: Swechhya/R-actions@v1.1
+      with:
+        action: 'all'
```

Comment:
R builds and checks tests are failing in Github. Do you still include this file?